WordPressKit/ReaderPostServiceRemote+RelatedPosts.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+
+extension ReaderPostServiceRemote {
+
+    /// Returns a collection of RemoteReaderSimplePost
+    /// This method returns related posts for a source post.
+    ///
+    /// - Parameter postID: The source post's ID
+    /// - Parameter siteID: The source site's ID
+    /// - Parameter success: Called when the request succeeds and the data returned is valid
+    /// - Parameter failure: Called if the request fails for any reason, or the response data is invalid
+    public func fetchRelatedPosts(for postID: Int,
+                                  from siteID: Int,
+                                  success: @escaping ([RemoteReaderSimplePost]) -> Void,
+                                  failure: @escaping (Error?) -> Void) {
+
+        let endpoint = "read/site/\(siteID)/post/\(postID)/related"
+        let path = self.path(forEndpoint: endpoint, withVersion: ._1_2)
+        let parameters = [
```

Comment:
Should we include the constants in as optional arguments just in case we decide to change this value in the future?

Reply:
Fixed! f6360d9